js/editTask.js

```diff
@@ -0,0 +1,36 @@
+import { CLICK_EVENT, EDIT, COMPLETE, DELETE_TODO } from "/js/constants.js";
+import saveButtonActionTodo from "/js/utility/eTSaveBtnAct.js";
+import deleteButtonActionTodo from "/js/utility/eTDelBtnAct.js";
+import checkButtonActionTodo from "/js/utility/eTChkBtnAct.js";
+
+export default function editTask(element, taskList, id) {
+  element.querySelector(`[data-job="${DELETE_TODO}"]`).style.display = "none";
+  element.querySelector(`[data-job="${EDIT}"]`).style.display = "none";
+  element.querySelector(`[data-job="${COMPLETE}"]`).style.display = "none";
+
+  element.querySelector(".text").contentEditable = true;
+  element.querySelector(".text").focus();
```

Comment:
Can we have a const declare for `the element.querySelector(".text")` and use that instead?

Reply:
I think it's not possible since the value of the element varies on each call of editTask function.

Reply:
I was talking about this. 
```suggestion
  const textElement = element.querySelector(".text");
  textElement.contentEditable = true;
  textElement.focus();
```

Reply:
oh!! sorry for the confusion. I thought you would like me to add those lines to the 'constants.js' file. done!